components/lifter.py

```diff
@@ -46,8 +45,8 @@ def setup(self):
         self.motor.setNeutralMode(ctre.NeutralMode.Brake)
 
         self.motor.overrideLimitSwitchesEnable(False)
-        self.motor.configReverseLimitSwitchSource(ctre.talonsrx.TalonSRX.LimitSwitchSource.FeedbackConnector, ctre.talonsrx.TalonSRX.LimitSwitchNormal.NormallyOpen, deviceID=0, timeoutMs=10)
-        self.motor.configForwardLimitSwitchSource(ctre.talonsrx.TalonSRX.LimitSwitchSource.Deactivated, ctre.talonsrx.TalonSRX.LimitSwitchNormal.Disabled, deviceID=0, timeoutMs=10)
+        self.motor.configReverseLimitSwitchSource(ctre.TalonSRX.LimitSwitchSource.FeedbackConnector, ctre.TalonSRX.LimitSwitchNormal.NormallyOpen, deviceID=0, timeoutMs=10)
+        self.motor.configForwardLimitSwitchSource(ctre.TalonSRX.LimitSwitchSource.Deactivated, ctre.TalonSRX.LimitSwitchNormal.Disabled, deviceID=0, timeoutMs=10)
```

Comment:
Not going to refer to them as `ctre.LimitSwitchSource` and `ctre.LimitSwitchNormal`?

Reply:
TBH we should probably remove this entirely until we work out what the whole limit switch situation is.